src/documentation/common.md

```diff
@@ -0,0 +1,59 @@
+# Shared specifications for all converters
+This document explains the shared parts among all the converters when converting a 
+data set from a given format into a [NTFS](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md#physical_modestxt-requis) dataset.
+
+## Data prefix
+The construction of NTFS objects IDs requires, for uniqueness purpose, that a unique 
+prefix (specified for each source of data as an addtional parameter to each converter)
+needs to be included in every object's id.
+
+Prepending all the identifiers with a unique prefix ensures that the NTFS identifiers are unique accross all the NTFS datasets. With this assumption, merging two NTFS datasets can be done without worrying about conflicting identifiers.
+
+This prefix should be applied to all NTFS identifier except for the physical_mode identifiers that are standarized and fixed values. Fixed values are described in the [NTFS specifications](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md#physical_modestxt-requis)
```

Comment:
you can write `physical mode` instead of `physical_mode`

and except for commercial mode.